cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -241,6 +241,12 @@ func checkFlags(onGCE bool) error {
 		return nil
 	}
 
+	// Check if the gcloud credentials are available.
```

Comment:
Nit, can you tweak the comment to instead say something like:

// Check if gcloud credentials are available and if so, skip checking the GCE VM service account scope.